omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js

```diff
@@ -363,6 +363,14 @@ function PatientSearchWidget(configuration){
                             }
                         });
                     }
+                    else if (column.type === 'location') {
+                        jq.each(patient.identifiers, function(index, identifier) {
```

Comment:
Instead of just looping through all the identifiers and picking the first one, let's set it so it is configured to match a specific identifier type by uuid, like how the identifier functionality works.

for example, we'd want to be able set a value to specific the identifier type like we do when displaying an identifier:

https://github.com/PIH/openmrs-config-zl/blob/master/configuration/pih/pih-config-haiti-hiv.json#L81 (where in this case "139766e8-15f5-102d-96e4-000c29c2a5d7" is the uuid column of the HIVEMR-V1 in the patient_identifier_type table)

Code would likely be similar to the code in the link below, although I think it can be simpler, because I don't think we need to support asking for multiple locations, we can assume just a single uuid and skip all the logic around "isMultipleIdentifierTypes":

https://github.com/openmrs/openmrs-module-coreapps/blob/master/omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js#L313